devfile.yaml

```diff
@@ -0,0 +1,53 @@
+metadata:
+  name: che-docs-in-che
+projects:
+  - name: che-docs
+    source:
+      location: 'https://github.com/eclipse/che-docs.git'
+      type: git
+components:
+  - mountSources: true
+    endpoints:
+      - name: che-docs-web-server
+        attributes:
+          public: 'true'
+          protocol: http
+          path: /
+        port: 4000
+      - name: che-docs-livereload-server
+        attributes:
+          public: 'true'
+          protocol: http
+          path: /
+        port: 35729
+    memoryLimit: 1Gi
+    type: dockerimage
+    image: 'eclipse/che-docs:latest'
+    alias: che-docs-dev
+  - id: eclipse/che-theia/next
+    type: cheEditor
+  - id: joaompinto/vscode-asciidoctor/latest
+    type: chePlugin
+apiVersion: 1.0.0
+commands:
+  - name: build che-docs
+    actions:
+      - workdir: /projects/che-docs/src/main
+        type: exec
+        command: >-
+          cd /projects/che-docs/src/main && mkdir -p _site && jekyll build
```

Comment:
Sorry, you already set up working dir, I think you don't need extra command: `cd /projects/che-docs/src/main` &&